packages/artifact/src/internal/shared/artifact-twirp-client.ts

```diff
@@ -80,11 +80,13 @@ class ArtifactHttpClient implements Rpc {
       try {
         const response = await operation()
         const statusCode = response.message.statusCode
-        debug(`[Response] ${response.message.statusCode}`)
-        debug(JSON.stringify(response.message.headers, null, 2))
+        const body = await response.readBody()
+        debug(`[Response] - ${response.message.statusCode}`)
+        debug(`Headers: ${JSON.stringify(response.message.headers, null, 2)}`)
+        debug(`Body: ${body}`)
```

Comment:
I believe we may need to remove these lines.

Reply:
I actually added them to assist in debugging customer workflows. If they run into an issue, they can turn on debug logs and this can potentially help figure out what's wrong